custom_components/unifi_insights/api/base.py

```
@@ -259,7 +274,7 @@ async def _handle_response(
 
         if status >= HTTPStatus.BAD_REQUEST:
             raise UniFiResponseError(
-                f"API error: {response_text}",
+                f"API error (status {status})",
                 status_code=status,
                 response_body=response_text,
             )
```

Ruff (0.15.9)

[warning] Avoid specifying long messages outside the exception class

(TRY003)